pages/common/http-server.md

```diff
@@ -0,0 +1,36 @@
+# http-server
+
+> Simple static HTTP server to serve static files.
+> More information: <https://github.com/http-party/http-server>.
+
+- Start a HTTP server listening on the default port to serve the current directory:
+
+`http-server`
+
+- Start a HTTP server on a specific port to serve files from a specific directory:
+
+`http-server {{path/to/directory}} --port {{port}}`
+
+- Start a HTTP server using basic authentication:
+
+`http-server --username {{username}} --password {{password}}`
+
+- Start a HTTP server with directory listings disabled:
+
+`http-server -d {{false}}`
+
+- Start a HTTP server on the default port with SSL using the specified certificates:
```

Comment:
Wording is technically wrong here.
```suggestion
- Start a HTTPS server on the default port using the specified certificates:
```
I don't like the fact that it uses the `--ssl` argument (since SSL was replaced by TLS ages ago), but there's nothing we can do about that.